spec/classes/agent_spec.rb

```diff
@@ -23,18 +23,17 @@
     end
   end
 
-  context 'token => 1234, files_config_enabled => true' do
+  context 'token => 1234' do
     let(:facts) { FACTS }
     let(:params) {{
-      :token => '1234', :files_config_enabled => true
-    }}
+      :token => '1234'    }}
     it do
       should compile.with_all_deps
       should contain_class('opsmatic::debian')
       should contain_package('opsmatic-agent').with(
         'ensure' => 'present')
       should contain_file('/etc/opsmatic-agent.conf').with(
-        'content' => /paths_ignore = \[\]\nfiles_config_enabled = true\ngroups = ""/)
+        'content' => /host_alias = ""\ngroups = ""/)
```

Comment:
if we're going to include `host_alias` and `groups` in this PR, we should make the tests check that they work